test/translator/BUILD

```diff
@@ -53,6 +53,15 @@ cc_test(
   ]
 )
 
+cc_test(
+  name = "write_test",
+  srcs = [ "write_test.cc" ],
+  deps = [
+    "//lib/translator:write_lib",
+    "//lib/translator:common",
```

Comment:
common is not needed because it is included as a dependency to write_lib

Reply:
done!